_data/projects/learn.yml

```
@@ -0,0 +1,14 @@
+name: LearnAwesome
+desc: Humanity's universal learning map
+site: https://learnawesome.org
+tags:
+  - ruby
+  - rails
+  - css
+  - html
+  - javascript
+  - education
+  - learning
+upforgrabs:
+  name: up for grabs
```

This label needs to match the one used in the project:

```suggestion
  name: up-for-grabs
```